### PR TITLE
Create empty environments

### PR DESCRIPTION
**Pull request by dorzel**
create empty environments when no packages are supplied in create

### CODE REVIEW

**Comment by kalefranz** [comment]
Will definitely need at least one test in `test_create` for this.  Read through the tests that are currently there for examples and inspiration.

**Comment by dorzel** [comment]
@kalefranz all tests passed

File: conda/cli/common.py [hunk 1]
```diff
@@ -548,20 +548,21 @@ def names_in_specs(names, specs):
...
                 an environment.  Doing so allows you to call conda create without
                 explicitly providing any package names.
 
                 To set the provided packages, call conda config like this:
 
                     conda config --add create_default_packages PACKAGE_NAME
+
             """)
-        raise TooFewArgumentsError(msg, json)
+        sys.stdout.write(msg)
```
Comment (kalefranz) [inline conda/cli/common.py]: This usually ends up being just as bad as sys.exit.  I don't think we need this `check_specs` at all.

Comment (kalefranz) [inline conda/cli/common.py]: When you remove it, make sure you remove it everywhere it's used too.

File: conda/history.py [hunk 2]
```diff
@@ -89,13 +89,17 @@ def update(self):
                 return
             curr = set(linked(self.prefix))
             if last == curr:
+                # print a head when a blank env is first created to preserve history
+                if enter_or_exit == 'exit' and not self.parse():
```
Comment (kalefranz) [inline conda/history.py]: `self.parse()` is a pretty heavy function for just needing to know if there's anything in the history file.  I wonder if there's an easier way to get this information?

**Comment by kalefranz** [comment]
This PR can be merged by Monday for 4.2.0rc1 as is, but there's still a bit more work to do here and there I think.  One thing that I think we probably need is some additional unit tests for `History` now.

File: conda/cli/common.py (hunk 1 above)
Comment (dorzel) [inline conda/cli/common.py]: Yeah I didn't think check_specs was really necessary either after adding in empty envs. Looks like it's only used once, in cli/install. But shouldn't there be a message saying that an empty env will be created? That's pretty much all check_specs does.

File: conda/history.py (hunk 2 above)
Comment (dorzel) [inline conda/history.py]: changed to an os.stat implementation

**Comment by dorzel** [comment]
@kalefranz let me know if this looks good

**Pull request merged**